### PR TITLE
[docs] Don't mark client as server in agent docs

### PR DESCRIPTION
**Pull request by Sanskar531**
### Description

In agent docs, it shows that a client is configured as server. Please check https://developer.hashicorp.com/consul/docs/agent#client-node-with-multiple-interfaces-or-ip-addresses 

### Testing & Reproduction steps

Visit https://developer.hashicorp.com/consul/docs/agent#client-node-with-multiple-interfaces-or-ip-addresses  and see that client is configured like this:

```hcl
server = true
``` 

```json
"server": true
```

### Links

Linked issue #18523

### PR Checklist

* [ ] updated test coverage (not applicable)
* [x] external facing docs updated
* [ ] appropriate backport labels added (not applicable)
* [ ] not a security concern (not applicable)

### CODE REVIEW

**Comment by Sanskar531** [comment]
Hi all! I would be nice to get someone to review this : )

File: website/content/docs/agent/index.mdx [hunk 1]
```diff
@@ -383,7 +383,7 @@ The `client_addr` configuration specifies IP addresses used for HTTP, HTTPS, DNS
 
 ```hcl
 node_name = "consul-server"
```
Comment (blake) [inline website/content/docs/agent/index.mdx:385]:
```suggestion
node_name = "consul-client"
```

File: website/content/docs/agent/index.mdx [hunk 2]
```diff
@@ -406,7 +406,7 @@ advertise_addr = "{{ GetInterfaceIP \"en0\" }}"
 ```json
 {
   "node_name": "consul-server",
```
Comment (blake) [inline website/content/docs/agent/index.mdx:408]:
```suggestion
  "node_name": "consul-client",
```

**Review by blake — CHANGES_REQUESTED** [review]
Thanks for picking this up. I requested a small change to the node name in the agent config. Otherwise, LGTM.

**Pull request closed without merging**